docs/source/configuration/overview.mdx

```diff
@@ -110,177 +141,76 @@ One you have generated the schema configure your text editor. Here are some inst
-    # Defaults to true
-    allow_any_origin: true
-    # List of accepted origins
-    origins:
-      - https://studio.apollographql.com
-    # Set to true to add the `Access-Control-Allow-Credentials` header
-    allow_credentials: false
-    # The headers to allow.
-    # Defaults to the required request header for Apollo Studio: Content-Type
-    allow_headers: [ Content-Type ]
-    # Allowed request methods
-    # Defaults to GET, POST, OPTIONS.
-    methods: [ GET, POST, OPTIONS ]
-    # Which response headers should be made available to scripts running in the
-    # browser in response to a cross-origin request.
-    expose_headers:
-plugins:
-    # See below for plugins
 ```
 
-## Avoiding repetition using YAML anchors and aliases
+It can also listen on a Unix socket (not supported on Windows):
 
-You can avoid repeating sections of configuration multiple times in your configuration file using standard YAML anchors and aliasing techniques:
+```yaml title="configuration.router.yaml"
+server:
+  # absolute path to a Unix socket
+  listen: /tmp/router.sock
 
-```yaml {5,9} title="configuration.yaml"
-plugins:
-  apollo.headers:
-    subgraphs:
-      products:
-        - insert: &insert_custom_header
-            name: "custom-header"
-            value: "something"
-      reviews:
-        - insert: *insert_custom_header
 ```
 
-In this example, the `&insert_custom_header` anchor captures the `name` and `value` properties from the `products` subgraph's `insert` and re-uses the same properties in the `insert` of the `reviews` subgraph by referring to the `*insert_custom_header` alias.
+### Cross Origin Resource Sharing (CORS)
 
-## OpenTelemetry tracing
+The Apollo Router can configure [Cross Origin Resource Sharing policies](../cors) throughits configuration file.
 
-The Apollo Router supports [OpenTelemetry](https://opentelemetry.io/), with exporters for [Jaeger](https://www.jaegertracing.io/) and the [OpenTelemetry Protocol (OTLP)](https://aws-otel.github.io/docs/components/otlp-exporter) over HTTP or gRPC. The router generates traces that include the various phases of executing a GraphQL operation and associated dependencies.
+### OpenTelemetry tracing
 
-The router sends a `TraceParent` header to subgraphs so they can create and send their own spans under the same trace.
+The Apollo Router supports [OpenTelemetry](https://opentelemetry.io/), with exporters for
+[Jaeger](https://www.jaegertracing.io/) and the
+[OpenTelemetry Protocol (OTLP)](https://aws-otel.github.io/docs/components/otlp-exporter)
+over HTTP or gRPC. The router generates traces that include the various phases of executing
+a GraphQL operation and associated dependencies.
 
-If the router receives an incoming request that already contains a `TraceParent` header, it uses that header's trace ID instead of generating a random one.
+You can set it up in the configuration file's [telemetry section](../telemetry).
```

Comment:
The telemetry page has gone, point to `open-telemetry`

Reply:
done in f289eb3